class.photon.php

```diff
@@ -160,6 +160,9 @@ public static function filter_the_content( $content ) {
 				if ( preg_match( '#data-lazy-src=["|\'](.+?)["|\']#i', $images['img_tag'][ $index ], $lazy_load_src ) ) {
 					$placeholder_src = $placeholder_src_orig = $src;
 					$src = $src_orig = $lazy_load_src[1];
+				} elseif ( preg_match( '#data-lazy-original=["|\'](.+?)["|\']#i', $images['img_tag'][ $index ], $lazy_load_src ) ) {
+						$placeholder_src = $placeholder_src_orig = $src;
```

Comment:
I'm guessing that it's because @chaselivingston knows how much it drives you crazy ;-)

Fixing now